api/v1/tetragon/tetragon.proto

```diff
@@ -404,6 +408,7 @@ message KprobeArgument {
 	string cap_permitted_arg = 24; // Capabilities that are currently permitted in hexadecimal format.
 	string cap_effective_arg = 25; // Capabilities that are actually used in hexadecimal format.
 	KprobeLinuxBinprm linux_binprm_arg = 26;
+    KprobeNetDev net_dev_arg = 27;
```

Comment:
maybe indention here?

Reply:
Ah ```make format``` doesn't fix that automatically and because its a .proto file, VSCode doesn't really know what to do with it. I will fix and push.